include/swift/AST/Attr.h

```diff
@@ -1667,7 +1668,8 @@ class DifferentiableAttr final
   /// specified.
   FuncDecl *VJPFunction = nullptr;
   /// The differentiability parameter indices, resolved by the type checker.
-  IndexSubset *ParameterIndices = nullptr;
+  /// The bit stores whether the parameter indices have been computed.
+  llvm::PointerIntPair<IndexSubset *, 1, bool> ParameterIndicesAndBit;
```

Comment:
`nullptr` is not used because `nullptr` by itself is ambiguous:
* "This attribute has not been type-checked", vs
* "This attribute failing type-checking, no parameter indices could be resolved"

I can add a comment.

Reply:
Done.